model/src/main/java/de/evoila/cf/broker/model/context/Context.java

```diff
@@ -0,0 +1,174 @@
+        return organizationGuid;
+    }
+
+    public void setOrganizationGuid(String organizationGuid) {
+        this.organizationGuid = organizationGuid;
+    }
+
+    public String getOrganizationName() {
+        return organizationName;
+    }
+
+    public void setOrganizationName(String organizationName) {
+        this.organizationName = organizationName;
+    }
+
+    public String getSpaceGuid() {
+        return spaceGuid;
+    }
+
+    public void setSpaceGuid(String spaceGuid) {
+        this.spaceGuid = spaceGuid;
+    }
+
+    public String getSpaceName() {
+        return spaceName;
+    }
+
+    public void setSpaceName(String spaceName) {
+        this.spaceName = spaceName;
+    }
+
+    public String getInstanceName() {
+        return instanceName;
+    }
+
+    public void setInstanceName(String instanceName) {
+        this.instanceName = instanceName;
+    }
+
+    public String getClusterId() {
+        return clusterId;
+    }
+
+    public void setClusterId(String clusterId) {
+        this.clusterId = clusterId;
+    }
+
+    public String getNamespace() {
+        return namespace;
+    }
+
+    public void setNamespace(String namespace) {
+        this.namespace = namespace;
+    }
+
+    @JsonAnyGetter
+    public Map<String, Object> getAdditionalFields() {
+        return additionalFields;
+    }
+
+    @JsonAnySetter
+    public void setAdditionalFields(String key, Object value) {
+        if (additionalFields == null) {
+            additionalFields = new HashMap<>();
+        }
+
+        this.additionalFields.put(key, value);
+    }
+
+    public void validateContextObject() {
+        switch (this.getPlatform()) {
+            case PLATFORM_CLOUDFOUNDRY:
+                validateCloudfoundryContextObject();
+                break;
+            case PLATFORM_KUBERNETES:
+                validateKubernetesContextObject();
+        }
+
+    }
+
+    private void validateKubernetesContextObject() {
+        if (fieldIsPresent(this.getNamespace()) && fieldIsPresent(this.getClusterId())) {
+            log.debug("Received a valid Kubernetes context object.");
+        } else {
+            throw new IllegalArgumentException("A Kubernetes Context object should contain the fields namespace and clusterId.");
+        }
+    }
+
+    private void validateCloudfoundryContextObject() {
+        if (fieldIsPresent(this.getOrganizationGuid()) &&
+                fieldIsPresent(this.getOrganizationGuid()) &&
+                fieldIsPresent(this.getOrganizationName()) &&
+                fieldIsPresent(this.getSpaceGuid()) &&
+                fieldIsPresent(this.getSpaceName()) &&
+                fieldIsPresent(this.getInstanceName())
+        ) {
+            log.debug("Received a valid Cloudfoundry context object.");
+        } else {
+            throw new IllegalArgumentException("A valid Cloudfoundry context object should contain organization_guid, organization_name, space_guid, space_name and instance_name");
```

Comment:
Unchecked exception throw and no note about it anywhere. Who catches this?

Reply:
By Jackson